cookbook/security/pre_authenticated.rst

```diff
@@ -11,6 +11,14 @@ box, Symfony supports most authentication mechanisms.
 These requests are called *pre authenticated* requests because the user is already
 authenticated when reaching your application.
 
+.. caution::
+
+    :doc:`User impersonation </cookbook/security/impersonating_user>` is not
+    compatible with pre Authenticated firewalls. The reason is that
```

Comment:
@pasdeloup the reason is that we use a strange format for titles, but for regular text we don't mix upper and lower case.